config/locales/controller/ja.yml

```
@@ -0,0 +1,8 @@
+ja:
+  books:
+    create:
+      notice: "本の追加に成功しました。"
```

```suggestion
      notice: "%{name}の追加に成功しました。"
```

のようにパラメータ化すると、今後モデルが増えたときも再利用できます。
ただし、キー名はbooksではなくcommonのような名前に変更しましょう。

また、全体的にパラメータ化することで再利用できそうな翻訳がないか見直してみてください

---

翻訳用のymlファイルを一通り修正いたしました。
それともないviewsでの参照も変更いたしました。